charts/piraeus/values.yaml

```diff
@@ -83,6 +83,7 @@ operator:
     monitoringImage: quay.io/piraeusdatastore/drbd-reactor:v0.4.4
     kernelModuleInjectionImage: quay.io/piraeusdatastore/drbd9-bionic:v9.1.4
     kernelModuleInjectionMode: Compile
+    kernelModuleInjectionMountUsrSrc: ""
```

Comment:
According to the docs, only `true` and `false` are allowed. Shouldn't this be `true`? How does this even work? :thinking: